starcheck/src/starcheck.pl

```diff
@@ -108,6 +109,45 @@
     return state
 
 
+def get_now_time():
+    """
+    Return the current time in YYYY:DOY format
+    """
+    return DateTime().date
+
+
+def run_start_time(run_start, backstop_start):
```

Comment:
Call this `get_run_start_time` for consistency with the previous, and to avoid overloading `run_start_time` with two things, the variable name (in perl) and the function name in Python.  Then you can use `run_start_time` as the arg name here and avoid name changes.